src/app/static/src/app/components/baseline/Baseline.vue

```diff
@@ -5,7 +5,7 @@
                 <form>
                     <file-upload label="PJNZ"
                                  :valid="pjnz.valid"
-                                 :error="pjnz.error"
+                                 :error="pjnz.error || plottingMetadataError"
```

Comment:
Wasn't really sure where to raise any errors fetching metadata to the user, but since we're tying the fetch to the PJNZ upload, it seems to make sense here.